docs/source/testing/mocking.md

```diff
@@ -30,19 +30,97 @@ server.listen().then(({ url }) => {
 
-Mocking logic looks at the types expected by your schema's fields and returns values based on those types. For example, mocking will return a string where a string is expected, a number for a Int, etc. The shape of your mocked results should always be consistent with the shape of a real result from a successful GraphQL operation.
+The table below covers the default scalar types and the default mocked values returned for each type:
 
-When using mocks your existing resolvers are **ignored** by default. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below to see how to change this behavior.
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Type</th>
+      <th>Default Mock Value</th>
+    </tr>
+  </thead>
 
-For more sophisticated testing, you can customize your mocks to return user-specified data.
+<tbody>
+<tr >
+<td>
+
+##### `Int`
+
+</td>
+<td>
+
+Returns a random positive or negative integer.
+</td>
+</tr>
+
+
+<tr>
+<td>
+
+##### `String`
+
+</td>
+<td>
+
+Returns "Hello world".
+</td>
+</tr>
+
+
+<tr>
+<td>
+
+##### `Float`
+
+</td>
+<td>
+
+Returns a random positive or negative double-precision floating-point value.
+</td>
+</tr>
+
+
+<tr>
+<td>
+
+##### `Boolean`
+
+</td>
+<td>
+
+Randomly returns either `true` or `false`.
+</td>
+</tr>
+
+
+<tr>
+<td>
+
+##### `ID`
+
+</td>
+<td>
+
+Returns a randomized UUID containing a combination of integers and letters.
+</td>
+</tr>
+
+</tbody>
+</table>
+
+By default when using mocks, your existing resolvers are **ignored** in favor of `mocks`. To configure this behavior, see [Using existing resolvers with mocks](#using-existing-resolvers-with-mocks).
+
+>Note: If `typeDefs` has any [custom scalar types](../schema/custom-scalars/#providing-custom-scalars-to-apollo-server), you will need to specify what your server should return for those types. You can do this by creating a customized mock with resolvers for each custom scalar type, as described below.
 
 ## Customizing mocks
 
-The `mocks` option can be assigned to an object describing custom mocking logic, allowing you to specify the values to return for specific field types. The `mocks` object allows you to define functions for field types, which will be invoked when that field type is expected. 
+For more sophisticated testing, you can customize your mocks to return user-specified data.
+
+Instead of providing a boolean to the `mocks` option, you can provide an object that defines custom mocking logic. This enables you to specify values to return for different return types.
 
-By default, the functions in `mocks` will overwrite any currently defined resolvers. In the below example, both `hello` and `resolved` will return `Hello`.
+By default, the functions in `mocks` will take precedence over any currently defined resolvers. In the below example, both `hello` and `resolved` return `Hello`.
```

Comment:
Cut "will"

Reply:
For an abundance of clarity, I might say "both `Query.hello` and `Query.resolved`"